HACKING.md

```diff
@@ -489,6 +489,18 @@ oc annotate secret/additional-root-ca-cert \
 The description can be customized. All other fields must be
 exactly as is.
 
+### [OPTIONAL] Create splunk secret
+
```

Comment:
For consistency with the other sections, can we add some context here? At least let's mention that the secret is located in BitWarden.

Reply:
It would also be nice to show an example of what that secret looks like (without any secret data in the example, of course).